docs/content/usage/networkconnections.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Network Connections
+description: Using the Network Connections API
+status: New
+source: https://github.com/xedi/sdk-js/blob/master/src/Services/NetworkConnections.ts
+---
+
+import {CheckIcon} from '@primer/styled-octicons'
+
+## Introduction
+
+An Network Connection allows you to set up a connection between 2 mailboxes. Connection types include; FTP, XEDI, AS2.
```

Comment:
This is more a description of the entity rather than the API. It serves well as a Jargon intro, but not necessarily as the leading line for the API intro.

Reply:
Also, Grammar